tests/osaka/eip7692_eof_v1/eip5450_stack/test_code_validation.py

```diff
@@ -433,3 +434,52 @@ def test_rjumps_jumpf_nonreturning(
         possible_exceptions.append(EOFException.INVALID_NON_RETURNING_FLAG)
 
     eof_test(container=Container(sections=sections), expect_exception=possible_exceptions or None)
+
+
+def gen_stack_underflow_params() -> Generator[Tuple[Op, int], None, None]:
+    """Generate parameters for stack underflow tests."""
+    for op in sorted(valid_eof_opcodes):
+        if op.min_stack_height == 0:
+            continue
+        if op in (Op.EOFCREATE, Op.RETURNCODE):
+            continue
+        yield op, 0
+        if op.min_stack_height > 1:
+            yield op, op.min_stack_height - 1
+
+
+@pytest.mark.parametrize("spread", [0, 1, MAX_OPERAND_STACK_HEIGHT])
+@pytest.mark.parametrize("op,stack_height", gen_stack_underflow_params())
+def test_stack_underflow(eof_test: EOFTestFiller, op: Op, stack_height: int, spread: int):
+    """
+    Test EOF validation failing due to stack overflow
+    caused by the specific instruction `op`.
+    There is similar test in test_all_opcodes_stack_underflow().
```

Comment:
maybe call this one similarly then?

Reply:
`test_all_opcodes_variadic_stack_underflow()`?

Reply:
yup, if there's enough similarities ofc

Reply:
ah wait, without the `variadic`, unless there's a non-variadic version too? Anyhow, It is as good as it is now, don't worry about my comment too much, just a nitpick